src/st/no3/collision.c

```diff
@@ -0,0 +1,3 @@
+#include <common.h>
```

Comment:
Is this necessary (can this be included in collision.h instead)? It would be nice if the files were literally just a single include line.

The one place I can think of where this might not work is what I found with `AnimateEntity` in `cen` and `rwrp` where they don't expect a particular prototype to be included. I think those should be considered exceptions though and make those more complicated instead of the ones that follow a similar pattern.